test/testdrive/system-cluster.td

```diff
@@ -99,3 +104,12 @@ $ set-regex match=true|false replacement=<TRUE_OR_FALSE>
 
 > SHOW CLUSTER REPLICAS WHERE cluster = 'mz_system'
 mz_system r1 1 <TRUE_OR_FALSE>
+
+> SELECT COUNT(*) FROM mz_catalog.mz_cluster_replicas as r, mz_catalog.mz_clusters as c WHERE r.cluster_id = c.id and c.name = 'mz_support';
+0
+
+! ALTER CLUSTER mz_support SET (replication factor 1, size="2");
```

Comment:
```suggestion
! ALTER CLUSTER mz_support SET (replication factor 1, size '2');
```

Double quotes are identifiers in SQL, and while we do support them in this position, it's a quirk of how options work, and using strings (single quotes) is more standard!